### PR TITLE
Adapt setup to work on AWS

### PR DESCRIPTION
**Pull request by FinalAngel**
- splits requirements to develop and master
- works locally and on docker hub

### CODE REVIEW

File: requirements/master.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Django>=1.8.5
 django-cms>=3.1.3
 django-reversion>=1.9.3             # not a part of django-cms core
-djangocms-text-ckeditor>=2.6.0      # not a part of django-cms core
+djangocms-text-ckeditor>=2.6.0    # not a part of django-cms core
```
Comment (vxsx) [inline requirements/master.txt]: tsk tsk

**Comment by stefanfoulis** [comment]
why do the requirements need to be split?
Does it not make the tests less realistic if travis installs different requirements than the normal Dockerfile?

**Comment by FinalAngel** [comment]
@stefanfoulis on travis we want the latest releases to pass, If there is an error we need to fix addons ASAP.
On docker and locally, we want the latest develop version to be installed so `preview.django-cms.org` shows our latest efforts. Optionally you can use `make install REQUIREMENTS=master` to also install latest releases on your local env.
I still need to adapt the requirements to use all develop versions, once the setup is up and running.

**Comment by stefanfoulis** [comment]
makes sense :-)

:+1: 

as a later step it could be useful to use ``pip-compile`` (from ``pip-tools``) to test all the requirements. That would also detect impossible combinations of dependencies (e.g one package says ``html5lib<0.10`` and another ``html5lib>=0.999999``). ``pip`` alone does not see such inconsistencies.

**Comment by FinalAngel** [comment]
will keep that in mind @stefanfoulis thank you

**Pull request merged**